src/thing_list.c

```diff
@@ -3561,7 +3561,7 @@ struct Thing *get_nearest_object_at_position(MapSubtlCoord stl_x, MapSubtlCoord
                     result = thing;
                 }
             }
-            blk = get_map_block_at(x, y);
+            blk += 5;
```

Comment:
What does this do?

Reply:
Given the size of a map block structure, I can only guess that it goes to the next map block.

Reply:
The point about these rewrites is trying to understand what the code does, and then writing new functions that do the same thing but are easy to understand for other humans reading the code.

So, can you get some confidence in your guess, and then write the code in such a way that what it does obvious from the code. Failing that, if need be, add a comment to it.